.github/workflows/test.yml

```
@@ -8,6 +8,7 @@ on:
     branches: [main, develop]
   pull_request:
     branches: [main, develop]
+  merge_queue:
```

actionlint (1.7.11)

[error] unknown Webhook event "merge_queue". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)